### PR TITLE
fix(blog): change post idiom

### PR DESCRIPTION
**Pull request by LucasSavoia**
**What problem is this solving?**
The posts were with other language, and we need it in portuguese.

**How should this be manually tested?**

**Screenshots or example usage:**
image

### CODE REVIEW

**Comment by LucasSavoia** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by LucasSavoia** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

File: react/components/WordpressPage.tsx [hunk 1]
```diff
@@ -204,8 +204,25 @@ const WordpressPageInner: FunctionComponent<{ pageData: any }> = props => {
           dangerouslySetInnerHTML={{ __html: titleHtml }}
         />
         <p className={`${handles.postMeta} t-small mw9 c-muted-1`}>
-          <span>Posted {formattedDate} </span>
-          {author && <span> by {author.name}</span>}
+          <span>
+            <formattedMessage
```
Comment (SDBowen) [inline react/components/WordpressPage.tsx:208]: `FormattedMessage`

and needs the import.

File: react/components/WordpressPost.tsx [hunk 2]
```diff
@@ -213,7 +213,15 @@ const WordpressPostInner: FunctionComponent<{
           dangerouslySetInnerHTML={{ __html: titleHtml }}
         />
         <p className={`${handles.postMeta} t-small mw9 c-muted-1`}>
-          <span>Posted {formattedDate} in </span>
+          <span>
+            <FormattedMessage
+              id="store/wordpress-integration.wordpressPost.postedIn"
+              defaultMessage="Posted {formattedDate} by"
```
Comment (SDBowen) [inline react/components/WordpressPost.tsx:219]: "Posted {formattedDate} **in** "

File: messages/en.json [hunk 3]
```diff
@@ -71,5 +71,9 @@
   "admin/editor.wordpressBreadcrumb.title": "Wordpress Breadcrumb",
   "admin/editor.wordpressBreadcrumb.description": "Displays a breadcrumb specific to your VTEX blog pages, i.e. 'Blog Home / (category) / (title of post)' ",
   "store/wordpress-integration.wordpressPostNavigation.next": "Next Article >",
-  "store/wordpress-integration.wordpressPostNavigation.previous": "< Previous Article"
+  "store/wordpress-integration.wordpressPostNavigation.previous": "< Previous Article",
+  "store/wordpress-integration.wordpressPost.postedIn": "Posted {formattedDate} by",
+  "store/wordpress-integration.wordpressPost.byAuthor": "by {name}",
+  "store/wordpress-integration.wordpressPage.posted": "Posted {formattedDate}",
+  "store/wordpress-integration.wordpressPage.byAuthor": "by {name}"
```
Comment (SDBowen) [inline messages/en.json:78]: "Posted {formattedDate} **in** "
" by {name}"
"Posted {formattedDate} "
" by {name}"

**Review by SDBowen — CHANGES_REQUESTED** [review]
@LucasSavoia thanks for the PR!

Seems we lost some spacing. Should be able to add it to the translation messages.

Screenshot from 2021-06-01 18-19-51

**Review by SDBowen — APPROVED** [review]

**Pull request merged**